python/setup.cfg

```diff
@@ -0,0 +1,43 @@
+[metadata]
+name = reactive-papi
+description = reactive-platform: Reactive Platform flatbuffer api
+long_description = file: README.md
+url = https://www.reactiveexchange.com
+project_urls =
+    Source=https://github.com/reactivemarkets/platform-api
+    Tracker=https://github.com/reactivemarkets/platform-api/issues
+author = Reactive Platform
+maintainer = Reactivemarkets
```

Comment:
"Reactive Markets" for author and maintainer?

Reply:
The author and maintainer like `reactivemarkets/developers` which is the name from github.